### ISSUE TITLE
DataLoader time is always 0

### ISSUE THREAD

**Issue by kiedanski** [pytorch/kineto#610]
I run your example https://github.com/pytorch/kineto/blob/main/tb_plugin/examples/resnet50_profiler_api.py and I keep getting a 0ms spent on the DataLoder. I tried to add a slow collate_fn function to see if that changed but I still get 0.

Is this the expected behaviour?

**Comment by aaronenyeshi** [comment]
Thanks for reporting, some of the examples are not working, we will investigate.

**Comment by deepanker13** [comment]
Issue → PyTorch profiler not capturing Dataloader time and runtime. Always shows 0.
Code used → I have used the code given in official PyTorch profiler documentation ( [PyTorch documentation](https://pytorch.org/tutorials/intermediate/tensorboard_profiler_tutorial.html))

Hardware Used-> Nvidia AI100 gpu
PyTorch version-> 1.13.0+cu117
PyTorch tensorboard profiler version → 0.4.1
<img width="1495" alt="Screenshot 2023-01-10 at 12 56 58 AM" src="https://user-images.githubusercontent.com/121787544/211985416-bed8ab4e-f52b-46e4-aecf-56e95e1fa412.png">


hi @aaronenyeshi , can you please help me here

**Comment by uygnef** [comment]
@guotuofeng
I have same issue. Could you please help me ?

**Comment by lyon-v** [comment]
I have same issue. Could you please help me ?

**Comment by CheerUpBee** [comment]
Same here

**Comment by elch10** [comment]
This fixed if use for example `schedule=torch.profiler.schedule(wait=1, warmup=1, active=8)`
